### DISCUSSION TITLE
How export pyvista Animation?

### DISCUSSION THREAD

**Discussion by FranzTr** [pyvista/pyvista#4634] [Q&A]
Hi everyone,

I' would like import the pyvista result on my web page 
When I create a 3D static image with pyvista no problem (I use the function .export_gltf and then import on the web using three.js)

Now, I'm creating a 3D animation with pyvista.
How can I export the animation as a 3D object? Can I use .export_gltf of all the movie?
Or is there another way to export the scene?

Thanks
Francesco

**Comment by stla** [comment]
Can't you made a GIF ?

**Comment by AlexandreDabySeesaram** [comment]
Hi, 
Have you found a way to export the animated scene in 3D? I'd like to do exactly the same thing and haven't succeeded so far.

**Comment by stla** [comment]
If you want a GIF you can follow this example : <https://dev.pyvista.org/examples/01-filter/flying_edges.html#animate-barth-sextic>.

**Reply by AlexandreDabySeesaram** [reply]
I was thinking of a way to share the interactive animated scene, similarly to what can be achieved with static scenes in HTML for instance where there would be the possibility to navigate through the 3D animation.

**Reply by stla** [reply]
Can't you do that with a gltf? It's possible to export to gltf with PyVista.

**Reply by AlexandreDabySeesaram** [reply]
That would be perfect but so far I only succeeded in exporting static scenes to gltf ... Do you have an example?

**Reply by stla** [reply]
I'm pretty sure I've already succeeded to export a gltf with possible interaction. Unfortunately the laptop I used for that is broken, and so I lost the code.

**Reply by AlexandreDabySeesaram** [reply]
Nice to know it's possible! thanks!
